### ISSUE TITLE
Add Overlay path

### ISSUE THREAD

**Issue by Waidhoferj** [calpoly-csai/argo-editor-frontend#2]
Be able to add a path to another panorama via an Overlay.

**Comment by Waidhoferj** [comment]
<img width="235" alt="Screen Shot 2021-04-18 at 9 47 46 AM" src="https://user-images.githubusercontent.com/33008436/115153504-25042a80-a02b-11eb-937c-015ba7db1886.png">
The user should upload a video and choose a destination location. Submitting the path form should upload the video to the server and store the result as an action on the overlay in the following form:

```
{
type: "path"
video: "<link to resource on Cloudinary>"
destination: "<id of location destination in tour graph>"
}
```

**Comment by Waidhoferj** [comment]
You can update the tour graph using the `onUpdate` function passed to the Overlay. Adding an action looks something like this:

```
onUpdate(overlay => void (overlay.actions.push(<your actions object>)));
```

**Comment by Waidhoferj** [comment]
Fixed in #12
